api/groomerFavorites/favModel.js

```diff
@@ -0,0 +1,20 @@
+const db = require('../../data/db-config');
```

Comment:
@SarahMLawrence Did these routes work in Insomnia or Postman

Reply:
@broph123  The routes are not fully functional yet. I will be fixing any errors and testing the routes in Insomnia this evening.

Reply:
are these routes fixed and functioning?

Reply:
@cearc-sara Yes, these routes are fixed and functioning.